src/components/header.js

```diff
@@ -27,6 +27,9 @@ const Header = ({ siteTitle }) => (
           <Link className="page-link" to="/match">
             Match
           </Link>
+          <Link className="page-link" to="/question">
+            Question
+          </Link>
```

Comment:
Can we at least move `Player` over to the `Team` docs group so the navbar still fits on one line?

Reply:
Sure.